pkg/command/configure/resource_test.go

```diff
@@ -24,11 +24,11 @@ import (
 func TestGetOverlayYamlContentSource(t *testing.T) {
 	for _, tt := range []struct {
 		name              string
-		resourcesCMDFlags resourcesFlags
+		resourcesCMDFlags ResourcesFlags
 		expectedResult    string
 	}{{
 		name: "Knative Eventing",
-		resourcesCMDFlags: resourcesFlags{
+		resourcesCMDFlags: ResourcesFlags{
```

Comment:
You are making this struct public because you are using in different package? maybe a little comment...